homeassistant/components/xiaomi_aqara.py

```diff
@@ -225,13 +247,33 @@ def device_state_attributes(self):
         """Return the state attributes."""
         return self._device_state_attributes
 
-    def push_data(self, data):
+    @callback
+    def _set_unavailable(self, now):
+        """Set state to UNAVAILABLE."""
+        self._remove_unavailability_tracker = None
+        self._is_available = False
+        self.async_schedule_update_ha_state()
+
+    def _track_unavailable(self):
```

Comment:
Please add the `callback` annotation and add `async` in the naem: `_async_track_unavailable`.